tasks/docs.py

```diff
@@ -252,39 +258,29 @@ def _generate_infrahub_schema_documentation() -> None:
         print(f"Docs saved to: {output_label}")
 
 
-@task
-def config(context: Context):
-    _generate_infrahub_config_documentation()
-
-
 def _generate_infrahub_config_documentation() -> None:
     import jinja2
-    from pydantic import BaseModel
 
     from infrahub import config
 
-    sections: List[ConfigurationSection] = []
-
+    sections: list[ConfigurationSection] = []
     schema = config.Settings.model_json_schema()
+
+    print("Rendering doc for Infrahub config...")
+
+    # Compute the data from config.py
     for prop in schema["properties"]:
-        if prop not in ["logging", "storage"]:
-            print(prop)
-            section_ref = schema["properties"][prop]["allOf"][0]["$ref"]
+        if prop not in ["logging", "storage"]:  # TODO: Why would we remove logging and storage here?
```

Comment:
I don't remember why this was done, the only reason I can think of was that it was failing if we had this and I didn't want to spend time on it just then.

Reply:
Even though it doesn't look pretty we could use something like `config.SETTINGS.storage.fget.__name__` to avoid hardcoded strings here